src/wazuh_db/wdb_delta_event.c

```diff
@@ -86,6 +88,7 @@ bool wdb_insert_dbsync(wdb_t * wdb, struct kv const *kv_value, const char *data)
 
 bool wdb_modify_dbsync(wdb_t * wdb, struct kv const *kv_value, const char *data)
 {
+    const char * treat_as_null[] = {"", NULL};
```

Comment:
Is there a possibility that this variable has to be assigned with the same values as L35?

Reply:
Nope. L35 cause that "NULL" or "" will be replaced to db's null elements during insertions. Modified events use "NULL" as mask (meaning the field was unchanged), but "" will indicate that a field was changed for some value to empty, and this need to be replaced by a db null